lib/libprocstat/libprocstat.c

```diff
@@ -2505,7 +2505,7 @@ procstat_getosrel(struct procstat *procstat, struct kinfo_proc *kp, int *osrelp)
 
 #define PROC_AUXV_MAX	256
 
-#if __ELF_WORD_SIZE == 64
+#if !defined(__ILP32__) && !defined(__riscv)
```

Comment:
Seems like we'd really want some kind of analog then for 'has freebsd32'.  Maybe have an internal header file #define a new macro (HAVE_ELF32 or some such) and use that in all these places instead?  This expression seems somewhat fragile with it being duplicated in multiple places.